setup.py

```diff
@@ -105,7 +105,7 @@ def run(self):
     packages=find_packages("src"),
     zip_safe=False,
     install_requires=install_requires,
-    extras_require={"dev": dev_requires},
+    extras_require={"dev": dev_requires, "rabbitmq": ["amqp==2.6.0"]},
```

Comment:
This is funny, from the cloudbuild logs:

```
Step #2: + pip install -r /tmp/dist/requirements.txt
Step #2: Ignoring docker: markers 'extra == "dev"' don't match your environment
-- snip --
Step #2: Ignoring amqp: markers 'extra == "rabbitmq"' don't match your environment
```

Yet the correct amqp version (and the dev dependencies as well) are still installed later on. @BYK do you know where `/dist/requirements.txt` comes from?